src/moda_bitbuf.F

```diff
@@ -12,6 +19,16 @@ MODULE MODA_BITBUF
 	  USE MODV_NFILES
 #endif
 
+C>        @var maxbyt
```

Comment:
I think you can also do this without the var tag, see https://github.com/NOAA-EMC/UFS_UTILS/blob/develop/sorc/chgres_cube.fd/atmosphere.F90 for a good example.